File: fib.ts
function fib(n:number): number{
    let fibarr=[0,1];
    for(let i=2;i<n;i++){
        fibarr[i]=fibarr[i-1]+fibarr[i-2];
    }
    return fibarr[n-1]
}
console.log(fib(40));

File: fib.java
/**
 * fib
 */
public class fib {

    public static void main(String[] args) {
        int n=80;
        long[] fibarr=new long[n];
        fibarr[0]=0;
        fibarr[1]=1;
        for(int i =2;i<n;i++){
            fibarr[i]=fibarr[i-1]+fibarr[i-2];
        }
        System.out.println(fibarr[n-1]);
    }
}